pkg/cmd/dotgit/dotgit.go

```diff
@@ -28,6 +28,7 @@ func NewCmdDotGit() *cobra.Command {
 
 			$ vuls-data-update dotgit registry ls
 			$ vuls-data-update dotgit registry status ghcr.io/vulsio/vuls-data-db:vuls-data-raw-debian-security-tracker-api
+			$ vuls-data-update dotgit registry push ghcr.io/vulsio/vuls-data-db vuls-data-raw-debian-security-tracker-api vuls-data-raw-debian-security-tracker-api.tar.zst
```

Comment:
```suggestion
			$ vuls-data-update dotgit registry push ghcr.io/vulsio/vuls-data-db:vuls-data-raw-debian-security-tracker-api vuls-data-raw-debian-security-tracker-api.tar.zst
```

Other dotgit registry commands uses "repo:tag" form.  How about doing the same?